packages/connectivity/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="io.flutter.plugins.connectivity"
+  android:versionCode="1"
+  android:versionName="0.0.1">
+
+  <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="21" />
+</manifest>
```

Comment:
Yes, ACCESS_NETWORK_INFO.  In google3, we did not have AndroidManifest.xml files for plugins. So we had to add it to the app manifest. Corrected; thanks!